ply-core/ply-core.cabal

```diff
@@ -27,9 +27,9 @@ common common-deps
     , bytestring
     , cardano-binary
     , containers
-    , plutus-core
-    , plutus-ledger-api
-    , plutus-tx
+    , plutus-core         >= 1.6.0.0 && < 1.8
```

Comment:
Should work all the way up to 1.10 but I didn't do it as it would require bump to index state, which is something I was slightly uncomfortable with.